src/API/Signals.php

```diff
@@ -4,9 +4,7 @@
 
 namespace OpenTelemetry\API;
 
-use InvalidArgumentException;
-
-class Signals
+interface Signals
```

Comment:
Does psalm complain about class keyword? I must say that it's very strange pattern for me to use interface to store constants only

Reply:
No, but it did complain about unused private constructors in a few classes (where a class had been used to store constants, and private constructor used to stop it being instantiated).
We have a couple of interfaces already which are used just to store constants, so I've updated these ones to be consistent.